fixtures/aritmeticops.rb.native

```diff
@@ -1,361 +1,356 @@
-{
-    "status": "ok",
-    "language": "ruby",
-    "errors": [],
-    "ast": {
-        "@type": "module",
-        "file": {
-            "@end": {
-                "@type": "ast:Position",
-                "col": 5,
-                "line": 7
+{ '@type': "module",
+   '@role': [Unannotated],
+   file: { '@type': "begin",
+      '@role': [Unannotated],
+      '@end': { '@type': "ast:Position",
```

Comment:
Should be:
```yaml
'@pos':
   '@type': "uast:Positions"
   start:
      '@type': "uast:Position"
      line: ...
      col: ...
   end:
      '@type': "uast:Position"
      line: ...
      col: ...
```

Reply:
I'll take a look but remember that in some drivers (Python specifically) not all nodes have positional information so the SDK shouldn't skip nodes without it.